common/reviews/api/rush-lib.api.md

```diff
@@ -625,6 +628,7 @@ export interface IOperationRunnerContext {
 
 // @alpha (undocumented)
 export interface IOperationSettings {
+    allowCobuildOrchestration?: boolean;
```

Comment:
Names in the public API are similarly confusing.

Reply:
updated in [c0fd113](https://github.com/microsoft/rushstack/pull/4871/commits/c0fd113fc40764c6525cec84d7ac06eeae836b68)